### ISSUE TITLE
loss=nan

### ISSUE THREAD

**Issue by XUJIALIOSWALD** [fxia22/pointnet.pytorch#78]
Hello, I have followed the steps to install and download the dataset, but why the loss value is always NAN, during training has not been solved after reducing the learning rate. What is the problem?
